tests/foreman/destructive/test_leapp_satellite.py

```diff
@@ -49,3 +49,6 @@ def test_positive_leapp(target_sat):
     result = target_sat.execute('cat /etc/redhat-release | grep -Po "\\d"')
     # Check if RHEL was upgraded
     assert result.stdout[0] == str(orig_rhel_ver + 1), 'RHEL was not upgraded'
```

Comment:
Do we have an issue that's stopping to upgrade of the OS version with leapp ? asking because of this  `str(orig_rhel_ver + 1), 'RHEL was not upgraded'`